app/views/layouts/_cart_area.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="cart_area">
+  <div class="cart_link">
+    <%= link_to "#" do %>
+      <i class="fa fa-shopping-basket"></i>
+      <span>(2) Item</span>
+    <% end%>
```

Comment:
.

Reply:
chấm gì á em???

Reply:
Cái này a từ cái header of e, tách partials ra, tưởng e làm đúng rồi nên a k check lại